.github/workflows/on-pull-request.yml

```diff
@@ -1,14 +1,20 @@
 name: "Verify PR"
 
 on:
-  pull_request
+  schedule:
```

Comment:
Why do we need schedule?

Reply:
This is to have an additional signal of PaaS health from the point of view of a client repo with the integration-test-cloud. It can be integrated/replicated in the PaaS repo itself at some point if deemed stable enough

If there is a way to trigger only that one every day that is fine with me

Reply:
There is probably not much to gain if the integration test code is not prebuilt and reused, we are setting up everything and compiling anyway. We could save the deployment to PaaS, but that's not significant time wise probably.

However I am not sure what happens if the app is already deployed to PaaS and we run again. Is it deployed from scratch or ignored.

If redeployed from scratch then it helps with overall health check of PaaS. But does not help to see if the app we deployed last time is still running.

Reply:
The full cycle is performed here, with app creation, waiting for health, IAM population, setting sources, perform NPL calls, and takes down the whole system.